modules/apps/27-interchain-accounts/controller/types/msgs.go

```diff
@@ -75,12 +73,12 @@ func (msg MsgSendTx) ValidateBasic() error {
 		return sdkerrors.Wrapf(sdkerrors.ErrInvalidAddress, "failed to parse owner address: %s", msg.Owner)
 	}
 
-	if msg.TimeoutHeight.IsZero() && msg.TimeoutTimestamp == 0 {
-		return sdkerrors.Wrap(channelerrors.ErrInvalidTimeout, "msg timeout height and msg timeout timestamp cannot both be 0")
+	if err := msg.PacketData.ValidateBasic(); err != nil {
+		return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "invalid interchain account packet data")
```

Comment:
It is actually fine to wrap the returned err. We can assume most everything will return a registered SDK error